silk/assets/configuration/spec.yaml

```
@@ -0,0 +1,32 @@
+name: Silk
+files:
+- name: silk.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/http
+    - template: init_config/default
+  - template: instances
+    options:
+      - name: host_address
+        description: Host address of Silk instance.
+        required: true
+        value:
+          example: http://localhost:80
+          type: string
+      - name: enable_blocksize_statistics
+        description: |
+          Enable this option to collect Silk stats metrics tagged by blocksize.
+        value:
+          type: boolean
+          example: true
+          display_default: false
```

Examples should match the default

```suggestion
          example: false
```